dev/TEAL.md

```diff
@@ -625,6 +654,28 @@ Account fields used in the `acct_params_get` opcode.
 | `log` | write A to log state of the current application |
 | `block f` | field F of block A. Fail unless A falls between txn.LastValid-1002 and txn.FirstValid (exclusive) |
 
+### Box Access
+
+All box related opcodes fail immediately if used in a
+ClearStateProgram. This behavior is meant to discourage Smart Contract
```

Comment:
or a logicsig

Reply:
That's true, but not what I want to discuss in this paragraph.  All opcodes are documented as stateful or not, but the box opcodes have the additional restriction of not working in ClearState (for the reason mentioned). I think it would be confusing the mention logicsigs here, since the reason mentioned here has nothing to do with logicsigs.